src/model/presentation/Presentation.java

```diff
@@ -0,0 +1,14 @@
+package model.presentation;
+
+import model.ShapesName;
+
+import java.util.List;
+
+/**
+ * Created by Xiaxing SHI on 27/10/15.
+ */
+public interface Presentation {
+    void addShapes(List<Shape> shapes);
```

Comment:
Add javadoc with the fact that addShapes creates the specific shapeState according to the type of shapes given